src/refmt_impl.ml

```diff
@@ -62,11 +79,16 @@ let () =
     "-print-width", Arg.Int (fun x -> print_width := Some x), "<print-width>, wrapping width for printing the AST";
     "-heuristics-file", Arg.String (fun x -> heuristics_file := Some x),
     "<path>, load path as a heuristics file to specify which constructors are defined with multi-arguments. Mostly used in removing [@implicit_arity] introduced from OCaml conversion.\n\t\texample.txt:\n\t\tConstructor1\n\t\tConstructor2";
-  ]
-  (fun arg -> filename := arg)
-  "Reason: Meta Language Utility"
-  in
+    "-h", Arg.Unit (fun () -> print_help := true), " Display this list of options";
+  ] in
+  let () = Arg.parse options (fun arg -> filename := arg) usage in
+  let print_help = !print_help in
   let filename = !filename in
+  let _ =
+    if filename = "" || print_help then
```

Comment:
The filename will be "" if we pass -use-stdin. 

Should check if that option is set or not here.